extensions/HuggingFace/typescript/hf.ts

```diff
@@ -294,7 +293,7 @@ function constructOutput(response: TextGenerationOutput): Output {
 
   const output = {
     output_type: "execute_result",
-    data: data,
+    data: data.generated_text,
     execution_count: 0,
     metadata: metadata,
```

Comment:
Should have metadata here as well
```
const data = response.generated_text;
const metadata = _.omit(response, ["generated_text"]);
```

Reply:
In our case we don't need it because there's no other field beside `generated_text` in the response, but yea good idea for future proofing